### PR TITLE
Handle attachments that are too large

### PR DESCRIPTION
**Pull request by wolverian**
Tell the user in the upload UI how large attachments we support, and change the text color to red if the user tries to upload an attachment that is too large.

### CODE REVIEW

**Comment by wolverian** [comment]
Normal state:
screen shot 2017-04-07 at 14 59 09

**Comment by wolverian** [comment]
Error state:
screen shot 2017-04-07 at 14 59 21

**Comment by wolverian** [comment]
If any of the files in the multiselect file set is too large, the whole file set upload fails. Already uploaded files (in a previous upload action) are preserved (I hope).

**Comment by wolverian** [comment]
The UI changes back to the normal state from the error state when the user uploads a new file successfully. The error state does not block the user from submitting the form.

File: src/cljs/ataru/hakija/application_form_components.cljs [hunk 1]
```diff
@@ -385,19 +387,29 @@
...
       [:i.zmdi.zmdi-cloud-upload.application__form-upload-icon]
       [:span.application__form-upload-button-add-text (case language
                                                         :fi "Lisää liite..."
                                                         :en "Upload attachment..."
-                                                        :sv "Ladda upp bilagan...")]]]))
+                                                        :sv "Ladda upp bilagan...")]]
+     (let [file-size-info-text (case language
+                                 :fi "Tiedoston maksimikoko on 10 MB"
+                                 :en "Maximum file size is 10 MB"
+                                 :sv "BÖÖ BÄÄ GUU")]
```
Comment (akaivola) [inline src/cljs/ataru/hakija/application_form_components.cljs]: 😜

**Pull request merged**